### PR TITLE
Elasticsearch 6 support

### PR DESCRIPTION
**Pull request by notbakaneko**
updates to use Elasticsearch 6:

- `type: string` is now `type: text`
- `index: not_analyzed` fields are now `type: keyword`
- `laravel-elasticsearch` package very unrequired

blocked until we actually setup elasticsearch 6 <_<

### CODE REVIEW

File: config/elasticsearch.php [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 return [
     'hosts' => (array) env('ES_HOST', 'localhost:9200'),
-
-    'search_timeout' => get_float(env('ES_SEARCH_TIMEOUT')) ?? 5,
-    'search_connect_timeout' => get_float(env('ES_SEARCH_CONNECT_TIMEOUT')) ?? 0.5,
+    'connectionParams' => [
+        'client' => [
+            'timeout' => get_float(env('ES_SEARCH_TIMEOUT')) ?? 5,
+            'connect_timeout' => get_float(env('ES_SEARCH_CONNECT_TIMEOUT')) ?? 0.5,
```
Comment (nanaya) [inline config/elasticsearch.php]: the original idea was to allow longer timeouts for indexing =S

Comment (nanaya) [reply config/elasticsearch.php]: also the env name can use an update

Comment (notbakaneko) [reply config/elasticsearch.php]: elasticsearch doesn't wait for indexing to run before responding; responses get returned immediately unless `?refresh=wait_for` is set

Comment (notbakaneko) [reply config/elasticsearch.php]: although it still needs to wait for the parser and errors to output 🤔

Comment (nanaya) [reply config/elasticsearch.php]: and can still use name update because it's initially named for search function (`ES_SEARCH_...`)

**Comment by notbakaneko** [comment]
soon™️

**Pull request merged**